brainscore_language/benchmarks/syntaxgym/__init__.py

```diff
@@ -0,0 +1,44 @@
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_core.metrics import Score
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language import load_metric, benchmark_registry
+from brainscore_language.benchmarks.syntaxgym.sg_suite import _load_suite
+
+class SyntaxGymTSE(BenchmarkBase):
```

Comment:
what does TSE stand for?

Reply:
"Targeted Syntactic Evaluations".  Comment added to the class.